app/assets/src/components/views/discovery/mapping/CircleMarker.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import BasicPopup from "~/components/BasicPopup";
+import cs from "./circle_marker.scss";
+
+class CircleMarker extends React.Component {
+  render() {
+    const { size, onClick, hoverContent } = this.props;
+
+    const circleBody = (
+      <circle
+        cx="50%"
+        cy="50%"
+        // Don't let edges get cut off in the viewBox
+        r={Math.max(size - 4, 1) / 2}
+        onClick={onClick}
+        className={cx(cs.circle, hoverContent && cs.hoverable)}
+      />
+    );
+    return (
+      <svg
+        height={size}
+        viewBox={`0 0 ${size} ${size}`}
+        // Proper centering
+        style={{ transform: `translate(${-size / 2}px, ${-size / 2}px)` }}
```

Comment:
So, you get center at 50% but then you translate it back?

Reply:
So this was going off an example but I think the svg transform places the viewbox over the center, and then the circle center places the circle in the center of the viewbox. is there another way to do this?

Without the viewbox transform:
![Screen Shot 2019-04-18 at 11 02 58 AM](https://user-images.githubusercontent.com/5652739/56382276-e2a17c00-61cb-11e9-8f48-3b1d6d5b65e2.png)

Without the circle center:
![Screen Shot 2019-04-18 at 11 03 35 AM](https://user-images.githubusercontent.com/5652739/56382295-e8975d00-61cb-11e9-984f-81910114ea82.png)

Both:
![Screen Shot 2019-04-18 at 11 04 08 AM](https://user-images.githubusercontent.com/5652739/56382301-ecc37a80-61cb-11e9-9585-19e492af10da.png)